### ISSUE TITLE
Three.js example does not work in Python

### ISSUE THREAD

**Issue by mdamien** [pyodide/pyodide#918]
I translated this example to Python: https://threejs.org/docs/#manual/en/introduction/Creating-a-scene

```python
from js import THREE, window

scene = THREE.Scene()
camera = THREE.PerspectiveCamera(75, window.innerWidth / window.innerHeight, 0.1, 1000)

renderer = THREE.WebGLRenderer()
renderer.setSize(window.innerWidth, window.innerHeight)
document.body.appendChild(renderer.domElement)
```

But I get `can't redefine non-configurable property "id"`

**Comment by hoodmane** [comment]
My guess is that the error occurs in the last line: `document.body.appendChild(renderer.domElement)`.
Could you try commenting it out and see if the script works up to that point? If the bug is in the last line, maybe we can find an example of the issue that doesn't require the `THREE` dependence.

Also, how are you getting `THREE`? Is there a cdn?

**Comment by mdamien** [comment]
The minimal example to reproduce the error:

```python
from js import THREE, window

scene = THREE.Scene()
camera = THREE.PerspectiveCamera(75, window.innerWidth / window.innerHeight, 0.1, 1000)
```

I'm getting Three via a CDN yes

**Comment by rth** [comment]
Note that in JS  example we have,
```js
const scene = new THREE.Scene();
```
so with pyodide to call the new operator it would be,
```py
scene = THREE.Scene.new();
```
cf https://pyodide.readthedocs.io/en/latest/type_conversions.html#javascript-from-python

However it's likely this will also not work due to the current limitation of the JS > Python interface https://github.com/iodide-project/pyodide/issues/788

**Comment by hoodmane** [comment]
I think this is closed by #1124 and #1126 and maybe some stuff. Please open a new issue if this still doesn't work.

**Comment by akaszynski** [comment]
If anyone is still interested, here's a few more pointers:

## `three_example.py`

```py
def display_scene():
    from pyodide import create_proxy

    from js import THREE, window

    scene = THREE.Scene.new();
    camera = THREE.PerspectiveCamera.new(75, window.innerWidth / window.innerHeight, 0.1, 1000)

    renderer = THREE.WebGLRenderer.new()
    renderer.setSize(window.innerWidth, window.innerHeight)

    geometry = THREE.BoxGeometry.new()
    material = THREE.MeshBasicMaterial.new(color="#00ff00")
    cube = THREE.Mesh.new( geometry, material )
    scene.add( cube )

    camera.position.z = 5
    renderer.render( scene, camera )

    def animate(*args):
        window.requestAnimationFrame( create_proxy(animate) )
        cube.rotation.x += 0.01
        cube.rotation.y += 0.01
        renderer.render( scene, camera )

    animate()

    return renderer

```

## `index.html`

```html
<!DOCTYPE html>
<html>
  <head>
    <title>Pyodide Test</title>
    <script src="https://cdn.jsdelivr.net/pyodide/v0.20.0/full/pyodide.js">
    </script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/three.js/r128/three.js">
    </script>
  </head>
  <body>
    <h1>Pyodide Test</h1>

    </ul>
    <script>
      async function main() {
        let pyodide = await loadPyodide();
        pyodide.runPython(
          await (await fetch("three_example.py")).text()
        );        
        let renderer = pyodide.runPython('display_scene()')
        document.body.appendChild( renderer.domElement );
      }

      main();
    </script>
  </body>
</html>
```
